modules/displaying-the-insights-status-in-the-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+[id="displaying-the-insights-status-in-the-web-console_{context}"]
+= Displaying the Insights status in the web console
+
+Insights repeatedly analyzes your cluster and you can display the status of identified potential issues of your cluster in the {product-title} web console. This status shows the number of issues in the different categories and, for further details, links to the reports in the {cloud-redhat-com}.
+
+.Prerequisites
+
+* Your cluster is registered in the {cloud-redhat-com}.
+* Remote health reporting is enabled, which is the default.
+* You are logged in to the {product-title} web console.
+
+.Procedure
+
+. Navigate to *Home* -> *Dashboards* in the {product-title} web console.
```

Comment:
The navigation should be `Home -> Overview`

Reply:
Fixed.